.github/workflows/eamxx-gh-clang-format.yml

```diff
@@ -105,19 +105,44 @@ jobs:
                "execute \`git apply <patch-file>\` from your local E3SM " \
                "branch." >> "${GITHUB_STEP_SUMMARY}"
           echo "Then commit the changes and push to your remote branch." >> "${GITHUB_STEP_SUMMARY}"
+          echo "" >> "${GITHUB_STEP_SUMMARY}"
+          echo "For diffs with ***fewer than 10 files***, we also include the text of " \
```

Comment:
Some files may still have A LOT of diffs, especially the first time the files undergo clang-format... Maybe you could just run `head -N 50 <diff_file>` (or 100, or whatever many lines you're comfortable with)?

Reply:
Do you think it's too much to ask that the user downloads the patch and looks at it? I think having a list of files that need fixing is plenty enough for the summary...